test/unit/test_lapack_tile/test_potrf.h

```diff
@@ -29,11 +29,10 @@ using namespace testing;
 
 template <class T, Device D, bool return_info>
 void testPotrf(const blas::Uplo uplo, const SizeType n, const SizeType extra_lda) {
+  std::function<T(const TileElementIndex&)> el_a, res_a;
```

Comment:
minor thing: why do you prefer it here? IMHO it is better to have it near its initialization.